Engine/Formatter.cs

```diff
@@ -12,6 +12,59 @@ namespace Microsoft.Windows.PowerShell.ScriptAnalyzer
     /// </summary>
     public class Formatter
     {
+        /// <summary>
+        /// Format a powershell script.
+        /// </summary>
+        /// <param name="scriptDefinition">A string representing a powershell script.</param>
+        /// <param name="settings">Settings to be used for formatting</param>
+        /// <param name="range">The range in which formatting should take place.</param>
+        /// <param name="runspace">The runspace entrance into the powershell engine.</param>
+        /// <param name="writer">The writer for operation message.</param>
+        /// <returns></returns>
+        public static string Format(
+            string scriptDefinition,
+            Settings settings,
+            Range range,
+            System.Management.Automation.Runspaces.Runspace runspace,
+            IOutputWriter writer)
+        {
+            // todo implement notnull attribute for such a check
+            ValidateNotNull(scriptDefinition, "scriptDefinition");
+            ValidateNotNull(settings, "settings");
+
+            Helper.Instance = new Helper(runspace.SessionStateProxy.InvokeCommand, writer);
+            Helper.Instance.Initialize();
+
+            var ruleOrder = new string[]
+            {
+                "PSPlaceCloseBrace",
+                "PSPlaceOpenBrace",
+                "PSUseConsistentWhitespace",
+                "PSUseConsistentIndentation",
+                "PSAlignAssignmentStatement",
+                "PSUseCorrectCasing"
+            };
+
+            var text = new EditableText(scriptDefinition);
+            foreach (var rule in ruleOrder)
+            {
+                if (!settings.RuleArguments.ContainsKey(rule))
+                {
+                    continue;
+                }
+
+                var currentSettings = GetCurrentSettings(settings, rule);
+                ScriptAnalyzer.Instance.UpdateSettings(currentSettings);
+                ScriptAnalyzer.Instance.Initialize(runspace, writer, null, null, null, null, true, false, null, false);
+
+                Range updatedRange;
+                bool fixesWereApplied;
+                text = ScriptAnalyzer.Instance.Fix(text, range, out updatedRange, out fixesWereApplied);
```

Comment:
the above `bool fixesWereApplied` line can be inlined just by using `out bool fixesWereApplied` here